device/device.go

```diff
@@ -43,3 +60,109 @@ func Init(typeFlag string) {
 		return
 	}
 }
+
+// New triggers select repository methods and initializes a new flasher
+func New(device string) (Flasher, error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 18 of function New() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->